test/fixture/api.js

```diff
@@ -1,6 +1,17 @@
+function sleep (ms) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, ms)
+  })
+}
+
 module.exports = {
   path: '/test_api',
-  handler (req, res) {
+  async handler (req, res) {
+    const query = new URL(req.url, 'http://localhost:3000').query
```

Comment:
Tip: We could also use [requrl](https://github.com/nuxt-contrib/requrl/) :)

Reply:
thank you 

I don't developed js backend so I am pretty clueless on what the current meta is